application/core/LSHttpRequest.php

```diff
@@ -42,6 +42,8 @@ class LSHttpRequest extends CHttpRequest
     /** @var array<string,mixed>|null the request query parameters (name-value pairs) */
     private $queryParams;
 
+    protected $_csrfToken;
```

Comment:
Why move to protected ? It's private in parent.

Reply:
In case someone wants to extend the LS class, desn't need to redeclare it.